app/models/book_author.rb

```diff
@@ -0,0 +1,4 @@
+class BookAuthor < ActiveRecord::Base
```

Comment:
I don't think we need a `through` table here... unless you're thinking a book *should* have many authors? For me, it makes more sense to have `book belongs_to :author`, and `author has_many :books`. Having a through table seems like premature normalization, and I don't see us needing to reference the BookAuthor relationship *directly*.